lib/engine/game/g_1880/game.rb

```diff
@@ -310,7 +311,7 @@ def event_stock_exchange_reopens!
         def event_token_cost_doubled!
           @log << "-- Event: #{EVENTS_TEXT['token_cost_doubled'][1]} --"
           @corporations.each do |c|
-            c.tokens.select(&:unused).each { |t| t.cost = t.cost * 2 }
+            c.tokens.reject(&:used).each { |t| t.price = t.price * 2 }
```

Comment:
Why reject instead of select?

Reply:
unused and cost aren't defined in for Token. changed it to rejecting used.
I could instead select ones where token.city = nil, but this seems cleaner.